src/server/common/helpers/logging/logger-options.js

```diff
@@ -1,16 +1,17 @@
 import ecsFormat from '@elastic/ecs-pino-format'
 
 import { config } from '~/src/config'
+import { redactedUploadContext } from '~/src/server/common/helpers/logging/logging-context-upload-redaction'
 
 const isDevelopment = config.get('isDevelopment')
-const redactionPaths = [
+let redactionPaths = [
```

Comment:
can stay as const. Your not changing the type, just the contents. Eslint should be telling you this

Reply:
I had to have it as a let when using concat as it creates a new array. Since I can use push then no such issue